spec/spec_helper_acceptance.rb

```diff
@@ -24,6 +24,16 @@
         # CentOS has epel-release package in Extras, enabled by default
         shell('yum -y install epel-release')
       end
+      puts "platform is #{host['platform']}"
+      if host['platform'] =~ /windows/
+        require 'winrm'
+        include Serverspec::Helper::Windows
+        include Serverspec::Helper::WinRM
```

Comment:
Making `is_cygwin: false` change results in Puppet making it to these two includes.  I don't see those helpers defined here: https://github.com/mizzy/serverspec/tree/master/lib/serverspec/type

Don't think those includes are necessary unless some they come from somewhere else

```
An error occurred in a `before(:suite)` hook.
Failure/Error: include Serverspec::Helper::Windows
NameError:
  uninitialized constant Serverspec::Helper::Windows
  Did you mean?  Windows
                 PSWindows

# ./spec/spec_helper_acceptance.rb:30:in `block (3 levels) in <top (required)>'
# ./spec/spec_helper_acceptance.rb:22:in `each'
# ./spec/spec_helper_acceptance.rb:22:in `block (2 levels) in <top (required)>'
```